### PR TITLE
Ignore Zope dependency

### LINKED ISSUE #98: Package hiding others
Not sure how this can actually be solved:

We have plenty of packages that depend on `Zope2`, that is fine, and as for `import` statements and such, with the mappings, all is good.

But since `Zope` is no longer on version 2 but rather on 5, the package got renamed to `Zope`.

Now if a package depends on `zope.schema` and `zope.interface` but also on `Zope` itself, `z3c.dependencychecker` fails to recognize that `zope.interface` and `zope.schema` are missing from `setup.py` since the package depends on `Zope` 💥 😱 

Since in this case [Zope](https://github.com/zopefoundation/Zope/tree/master/src) only is an umbrella and has no direct code, could we single it out in `z3c.dependencychecker`? 🍀 

And whenever another package does the same (let's say `Plone` 😅 ) we could add a key on `pyproject.toml` configuration that allows to list umbrella packages that provide no code on their own namespace? 🤔 

Ideas?

### PR DESCRIPTION
**Pull request by gforcada**
Fixes #98

### CODE REVIEW

File: z3c/dependencychecker/package.py [hunk 1]
```diff
@@ -121,6 +121,8 @@ def get_required_dependencies(self):
         this_package = self._get_ourselves_from_working_set()
         requirements = this_package.requires()
         for requirement in requirements:
+            if requirement.project_name == "Zope":
+                continue
```
Comment (reinout) [inline z3c/dependencychecker/package.py:125]: Would a `logger.debug("Ignoring 'Zope' requirement")` or so be a good idea?

**Review by reinout — APPROVED** [review]
One logging suggestion, but I leave it to you to see how important it is to add it :-)

File: z3c/dependencychecker/package.py (hunk 1 above)
Comment (gforcada) [reply z3c/dependencychecker/package.py:126]: Should we do the same on the extras requirements? As we do it within a list comprehension it is a bit trickier to do it though :-/

Comment (reinout) [reply z3c/dependencychecker/package.py:126]: Ouch, yes, that's trickier.
Well, I guess Zope will normally be a normal requirement, not somewhere in the extras? Fine enough if the main usecase is being logged.

Comment (gforcada) [reply z3c/dependencychecker/package.py:126]: Yes, that's what I though as well, but wanted to double check with you 😄

**Comment by gforcada** [comment]
@reinout could we get this merged and released? 😄 🤞🏾 🍀

**Comment by reinout** [comment]
`2.9` is out now.

I'm totally fine with you merging&releasing finished branches, btw. I think you've got enough rights on pypi for that?

**Comment by gforcada** [comment]
Yes, I think so,but as it'a been a while I wanted to ask confirmation first 😃

And thanks for the release 👍

**Pull request merged**